packages/widgets-manager/src/grid.ts

```diff
@@ -85,12 +155,30 @@ export class GridManager {
     cell.apps = positionStrategy.mountStrategy().add(app, cell.apps);
   }
 
+  // instead of create a new cell with need to map the closet cell that match on at least one axis x or y
+  // the complete refactor it's going to be added in the next widget manager release
   _addToNewCell(app: App, id: string, positionStrategy: PositionStrategy) {
     if (positionStrategy.shouldAddNewPosition()) {
       this.grid.cells.push(this._createNewCell(id, app, positionStrategy.mountStrategy()));
     }
   }
 
+  _addToClosetCell(app: App, id: string) {
+    const splitPosition = id.split('-')
+    // if cell is for kind center and layout is medium we push
+    // to most closest left cell or if of unknown we fallback to top-left
+    const cell = splitPosition[1] === 'center' ?
+      this._findCell(`${splitPosition[0]}-left`) : this.grid.cells[0] ;
+    // we know cell always exit but type say that could be undefined
+    if(cell) {
+      this._addToExistingCell(
+        app,
+        cell,
+        makePostionStrategy(app.settings.position.type)
+      );
+    }
+  }
```

Comment:
Is there a test for this method?

Reply:
For what I see `makePostionStrategy` doesn't have testing but its another method that is going to expire soon